src/algebra/group/defs.lean

```diff
@@ -796,7 +796,7 @@ by rw [mul_assoc, mul_right_inv, mul_one]
 @[simp, to_additive] lemma inv_mul_cancel_right (a b : G) : a * b⁻¹ * b = a :=
 by rw [mul_assoc, mul_left_inv, mul_one]
 
-@[priority 100, to_additive]
+@[priority 100, to_additive add_group.to_subtraction_monoid]
```

Comment:
Arguably this one should be fixed by adding a `subtraction` <-> `division` mapping to `to_additive`'s list of name translations.

Reply:
Wondering why it's not named `div_monoid` in the first place ... there it can take advantage of the mapping between `div` and `sub`.

Reply:
I failed to find a way to change `to_additive.tr` to achieve this automatically. If someone wants to do that, please push direct to this branch.

Otherwise, as it seems this problem only occurs once, I'm fine with doing it by hand here.

Reply:
The naive approach of duplicating

https://github.com/leanprover-community/mathlib/blob/ea1362f2bc50d121ecb2ff829335558a340e6a1b/src/algebra/group/to_additive.lean#L214

With the longer names doesn't work?

Reply:
The problem is that is still changes `monoid` to `add_monoid`.

Reply:
What if you use (untested)?
```
| is_comm ("division" :: "monoid" :: s) := "subtraction" :: add_comm_prefix is_comm "monoid" :: tr ff s
```

Reply:
Yes, that works.